src/styles/until/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$size-square:100px;
+$gap-square: 1px;
+$color-square:#eee;
+
+$max-width: 7 * $size-square + $gap-square * 7 + 10 * 2;
+
+$days:
+  "mon" ($size-square+$gap-square),
+  "tue" 2 * ($size-square+$gap-square),
+  "wed" 3 * ($size-square+$gap-square),
+  "thu" 4 * ($size-square+$gap-square),
+  "fri" 5 * ($size-square+$gap-square),
+  "sat" 6 * ($size-square+$gap-square);
+
```

Comment:
Great Job! 💪 But you need to fix some little things
https://imgur.com/a/yDpJYMX